nss-day-cohort-30/bangazon-site-suave-salmon
# abandoned product type report

Changes proposed in this request & reasons behind organizational or architectural decisions:
- added abandoned product type report view

Steps to test:
- git fetch --all
- checkout kc-branch
- git pull origin kc-branch
- run program, click "reports" in nav bar, then click "abandoned product type report"
- should see a list of up to 5 most frequently abandoned product types, and the number of open orders they occur on

Commit message:
- abandoned product type report 

Link to feature ticket:
- [#11](https://github.com/nss-day-cohort-30/bangazon-site-suave-salmon/issues/11)

@suave-salmon
